src/main/java/Deadlines.java

```diff
@@ -0,0 +1,18 @@
+class Deadline extends Task {
+
+    protected String deadline;
+
+    Deadline(String description, String deadline) {
+        super(description);
+        this.deadline = deadline;
+    }
+
+    String getDeadline() {
+        return this.deadline;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[D]%s (by: %s)", super.toString(), this.deadline);
```

Comment:
I like how you used the String format method to do this - this looks much cleaner than what I did!